config/crd/kustomization.yaml

```diff
@@ -10,4 +10,5 @@ resources:
 - bases/tower.ansible.com_ansibleschedules.yaml
 - bases/tower.ansible.com_ansibleinstancegroups.yaml
 - bases/tower.ansible.com_workflowtemplates.yaml
+- bases/tower.ansible.com_inventories.yaml
```

Comment:
I think we should be consistent in the naming, whether it is `ansibleinventories` or `inventories`. In this case, since the CRD is AnsibleInventory, I think we probably want this to be called `bases/tower.ansible.com_ansibleinventories.yaml`

Reply:
And it looks like that is what was used for the other kustomize files too:
- tower_v1alpha1_ansibleinventory-clone.yaml
- tower_v1alpha1_ansibleinventory-new.yaml

Reply:
+1...will patch up now